packages/@react-spectrum/tabs/docs/Tabs.mdx

```diff
@@ -0,0 +1,472 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/tabs';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/tabs/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Heading} from '@react-spectrum/text';
+import {Item, Tabs} from '@react-spectrum/tabs';
+```
+
+---
+category: Collections
+keywords: [tabs, collections]
+---
+
+# Tabs
+
+<p>{docs.exports.Tabs.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Tabs', 'Item']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/tabs/'}
+  ]} />
+
+## Example
+```tsx example
+import {Content} from '@react-spectrum/view';
+import {Text} from '@react-spectrum/text';
+
+<Tabs aria-label="History of Ancient Rome">
+  <Item title="Founding of Rome" key="FoR">
+    <Content marginTop="size-250" marginStart="size-125">
```

Comment:
It is not necessary for all of them (or really any of them since we can just use any old renderable node) but I found them useful for the margins and it kinda just made the code make more sense (delineates between the Tab and the Tab's contents).

When you say "build the margins into the tabs component" do you mean slapping these margin value into the tab panel div or adding them as a default and giving the user the ability to customize the tab panel's styles directly through some prop? I was thinking using `Content` would allow users to size/position their tab panels according to their needs, like if they wanted a negative margin so that the content could be offset to the left of the tablist

Reply:
Yeah I meant include on the tab panel itself. Seems like spectrum would have an opinion on what that margin would be. I guess we should ask them.

Reply:
Spectrum said they didn't have guidelines for the padding, but generally recommend no left/right padding and a top alignment that works with the product design. I would like to keep this `Content` bit so that end users have examples of how to adjust alignment, lemme know what you think